src/sortable.js

```diff
@@ -125,9 +131,41 @@ angular.module('ui.sortable', [])
                 opts[key] = patchSortableOption(key, value);
                 return;
               }
-
-              value = patchSortableOption(key, value);
-
+			  
+			  
+              /*
+              * If user defines ui-sortable-callback for @key and callback based option at the same time the html based ui-sortable-callback expression will be selected. (Overridden)
+              *	If user defined ui-sortable-callback for @key but callback expression is empty then option based @key callback will be selected.
+              * If user not defines a option for @key callback then html based ui-sortable-callback will be just selected.
+              * If user just defines option for @key callback then it will be attached.
+              * */
+              var attrKey = 'uiSortable'+key.substring(0,1).toUpperCase()+key.substring(1);
+              if(scope[key]!==undefined && scope[key] instanceof Function && attrs[attrKey] !== undefined && attrs[attrKey].length > 0) {
+                
+                var expression = scope[key]; //Scope variable can be changed on fly.
```

Comment:
What happens in case that the attribute callback was not provided up front but is set/passed at a later point of time? If I'm getting this right, we are not actually watching for changes, so setting a value at a later point of time (going from unset to pointing to a function) could have no effect.